### PR TITLE
feat(editor): make elements keyboard accessible

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to #173

* Make editor form fields accessible via keyboard
* Add `Enter` key handler to select focussed elements

Kapture 2023-04-28 at 13 26 36

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
@bpmn-io/human-task-orchestration feel free to [try out the demo](https://demo-173-editor-selection-keyb--camunda-form-playground.netlify.app/) and let me know what you think 👍

File: packages/form-js-editor/assets/form-js-editor-base.css [hunk 1]
```diff
@@ -265,6 +267,10 @@
   color: var(--color-context-pad-item-hover-fill);
 }
 
+.fjs-editor-container .fjs-context-pad-item:focus-visible {
+  outline-color: var(--color-children-selected-border);
```
Comment (vsgoulart) [inline packages/form-js-editor/assets/form-js-editor-base.css:271]: For me on Firefox on Linux I can only see the outline on the correct color if I do it like below. Is it the same on Firefox on Mac?
```suggestion
  outline: 2px solid var(--color-children-selected-border);
  border-radius: 2px;
```

**Review by vsgoulart — APPROVED** [review]

**Review by christian-konrad — APPROVED** [review]
Works well!

**Comment by RomanKostka** [comment]
looks and works good! 👍

One thing we may want to consider:
When i delete a component the next component is being 'active' but somehow the focus is lost and i am not able to delete this as well. By pressing TAB the next but one is being selected.

how to reproduce:
1. select a component
2. delete this component
3. press TAB

Expected behavior:
The selected element's delete icon should be focused

What currently happens:
The 2nd next element is being focused

**Comment by pinussilvestrus** [comment]
Nice finding @RomanKostka 👏 I'll investigate this!

**Comment by pinussilvestrus** [comment]
@RomanKostka it should be fixed now

**Pull request merged**